src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -165,14 +165,17 @@ Relative path: 'SonarLint.VisualStudio.ConnectedMode.dll'
 
 Referenced assemblies:
 - 'LibGit2Sharp, Version=0.26.0.0, Culture=neutral, PublicKeyToken=7cbde695407f0333'
+- 'Microsoft.VisualStudio.ComponentModelHost, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
+- 'Microsoft.VisualStudio.Shell.15.0, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
+- 'Microsoft.VisualStudio.Shell.Framework, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
```

Comment:
Not sure why these changes are here. Result of previous incorrect merge maybe?

Reply:
@rita-sonarsource the two added references are missing from the previous version of the file in `branch-sse-master3` - [here](https://github.com/SonarSource/sonarlint-visualstudio/blob/d05062f01d25347a302058f171c3a902b22f5ce9/src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt#L175).

I suspect it is the result of a bad merge. It won't cause a build failure in the PR CI run since that is not signing the assemblies -> the PR CI will be checking the other asmref baseline file (the one that contains the report for unsigned assemblies, which is correct).

To test, I've triggered a rebuild of the `branch-sse-master3` -> `master` PR with `ForceVsixSign` set to `true`. I think this will force the assemblies to be signed too, and I would expect the build to fail because the asmrefs are different.

Reply:
I ran rebuild with asmref and pushed an updated version to master3